File: Java Programming/Competitive Programming/Coding Challenges 14/program3.java
/*3. Write a program which accept distance in kilometre and convert it into meter. (1 kilometre = 1000 Meter)
Input: 5
Output: 5000

Input: 12
Output: 12000
*/

import java.util.Scanner;

class program3
{
    static int KMtoMeter(int iNo)
    {
        int iMeter = iNo * 1000;

        return iMeter;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        int iRet = 0;

        System.out.print("Enter distance: ");
        iValue = sobj.nextInt();

        iRet = KMtoMeter(iValue);

        System.out.print("Distance is meter is : " + iRet);

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 12/program3.cpp
/*3. Write a program which accept distance in kilometre and convert it into meter. (1 kilometre = 1000 Meter)
Input: 5
Output: 5000

Input: 12
Output: 12000
*/

#include <iostream>
using namespace std;

int KMtoMeter(int iNo)
{
    int iMeter = 0;

    for (int i = 1; i <= iNo; i++)
    {
        iMeter = iMeter + 1000;
    }

    return iMeter;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout << "Enter distance: ";
    cin >> iValue;

    iRet = KMtoMeter(iValue);

    cout << "Distance is meter is : " << iRet;

    return 0;
}